roles/lib_openshift/src/class/oc_group.py

```diff
@@ -0,0 +1,152 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+class OCGroup(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'group'
+
+    # pylint allows 5
+    # pylint: disable=too-many-arguments
+    def __init__(self,
+                 config,
+                 verbose=False):
+        ''' Constructor for OCGroup '''
+        super(OCGroup, self).__init__(config.namespace, config.kubeconfig)
+        self.config = config
+        self.namespace = config.namespace
+        self._group = None
+
+    @property
+    def group(self):
+        ''' property function service'''
+        if not self._group:
+            self.get()
+        return self._group
+
+    @group.setter
+    def group(self, data):
+        ''' setter function for yedit var '''
+        self._group = data
+
+    def exists(self):
+        ''' return whether a group exists '''
+        if self.group:
+            return True
+
+        return False
+
+    def get(self):
+        '''return group information '''
+        result = self._get(self.kind, self.config.name)
+        if result['returncode'] == 0:
+            self.group = Group(content=result['results'][0])
+        elif 'groups \"%s\" not found' % self.config.name in result['stderr']:
```

Comment:
nit(not_required): I'd recommend moving away from ``%s``'s and towards string formatting like so:

(assuming the \'s are required in the output check)

```python
elif 'groups \"{}\" not found'.format(self.config.name) in result['stderr']:
```